app/components/form/fields/DisksTableField.tsx

```diff
@@ -81,16 +87,31 @@ export function DisksTableField({ control }: { control: Control<InstanceCreateIn
         )}
 
         <div className="space-x-3">
-          <Button size="sm" onClick={() => setShowDiskCreate(true)}>
+          <Button
+            size="sm"
+            onClick={() => {
+              if (!disabled) {
+                setShowDiskCreate(true)
+              }
+            }}
+          >
```

Comment:
`Button` has a `disabled` prop that seems to work ok and gets you disabled styling too.